### ISSUE TITLE
error when I'm changing default import alias when creating pnpx create-next-app@latest

### ISSUE THREAD

**Issue by nicitaacom** [vercel/next.js#56780]
### Link to the code that reproduces this issue

https://github.com/nicitaacom/17_airbnb-clone/blob/production/airbnb-clone/app/components/ClientOnly.tsx

### To Reproduce

1. `pnpx create-next-app@latest`
2. choose yes until it ask you about import alias
3. when it ask you about 'Whould you like to change default import alias?' - select 'Yes' and type @/app/*
4. Got the error

### Current vs. Expected behavior

I wait not error - but I got the error and I need set up my import aliases in tsconfig.ts

### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

```bash
pnpx --no-install next info
/home/kali/.local/share/pnpm/store/v3/tmp/dlx-150353:
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/--no-install: Not Found - 404

This error happened while installing a direct dependency of /home/kali/.local/share/pnpm/store/v3/tmp/dlx-150353

--no-install is not in the npm registry, or you have no permission to fetch it.

No authorization header was set for the request.


Here should be pnpx next info --no-install
```

### Which area(s) are affected? (Select all that apply)

CLI (create-next-app)

**Comment by nicitaacom** [comment]
I don't think that it's relevant but defenitely requires check from someone else on next js latest

**Comment by marcoshernanz** [comment]
Thanks for the report. I could not reproduce this using `@/app/*` with `create-next-app@13.5.4`, the latest stable version when this was filed, or with current stable and canary releases. Project creation and `next build` both succeed with App Router, `src/`, and pnpm. Since the report does not include the error output or a related reproduction, and the original author no longer considers it relevant, I’m closing this. If this still occurs, please open a new issue with a current minimal reproduction and the complete error output.
